apps/condo/domains/resident/utils/accessSchema.js

```diff
@@ -0,0 +1,72 @@
+const { get, set } = require('lodash')
+
+const conf = require('@open-condo/config')
+const { getRedisClient } = require('@open-condo/keystone/redis')
+const { find } = require('@open-condo/keystone/schema')
+
+const _redisClient = getRedisClient('default', 'cache')
+const DEFAULT_CACHE_TTL_IN_MS = 60 * 60 * 1000 // 1 hour
+const CACHE_TTL_FROM_ENV = parseInt(conf['USER_RESIDENT_CACHING_TTL_IN_MS'])
+const CACHE_TTL_IN_MS = isNaN(CACHE_TTL_FROM_ENV) ? DEFAULT_CACHE_TTL_IN_MS : CACHE_TTL_FROM_ENV
+const DISABLE_USER_RESIDENT_CACHING = get(conf, 'DISABLE_USER_RESIDENT_CACHING', 'false').toLowerCase() === 'true'
```

Comment:
I think our common pattern of configuring stuff was JSON

I expected smth like: RESIDENT_CACHE_CONFIG=<JSON>